config/src/main/java/io/scalecube/config/ConfigRegistryImpl.java

```diff
@@ -152,24 +155,76 @@ public IntConfigProperty intProperty(String name) {
 
   @Override
   public int intValue(String name, int defaultValue) {
-    return intProperty(name).get(defaultValue);
+    return intProperty(name).value(defaultValue);
+  }
+
+  @Override
+  public DurationConfigProperty durationProperty(String name) {
+    return new DurationConfigPropertyImpl(name);
+  }
+
+  @Override
+  public Duration durationValue(String name, Duration defaultValue) {
+    return durationProperty(name).value(defaultValue);
+  }
+
+  @Override
+  public ListConfigProperty<String> stringListProperty(String name) {
+    return new ListConfigPropertyImpl<>(name, str -> Arrays.stream(str.split(",")).collect(Collectors.toList()));
+  }
+
+  @Override
+  public List<String> stringListValue(String name, List<String> defaultValue) {
+    return stringListProperty(name).value(defaultValue);
+  }
+
+  @Override
+  public ListConfigProperty<Double> doubleListProperty(String name) {
+    return new ListConfigPropertyImpl<>(name, str -> Arrays.stream(str.split(","))
+        .map(Double::parseDouble).collect(Collectors.toList()));
```

Comment:
Make ListConfigPropertyImpl constructor accept only value parser like:

```
new ListConfigPropertyImpl<>(name, Double::parseDouble);
```

So you can move part of shared code inside `ListConfigPropertyImpl`